source/rst/lss_examples.rst

```diff
@@ -1,36 +1,36 @@
 
-Methods of the Linear State Space Class
+Methods of the LinearStateSpace Class
 =======================================
 
 QuantEcon contains a class which has various methods for operating on
-Linear State Space Models (such models are explained in detail `in this
-lecture <https://lectures.quantecon.org/py/linear_models.html>`__).
+``LinearStateSpace`` Models (such models are explained in detail `in this
+lecture <https://lectures.quantecon.org/py/linear_models.html>`__)
 
 In this notebook, we will illustrate the methods that can be used on
-instances of this class.
+instances of this class
 
 To illustrate these methods, we will use Paul Samuleson's (1939)
-multiplier-accelerator model.
+multiplier-accelerator model
 
-Samuelson's (1939) multiplier-accelerator model
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Samuelson's (1939) Multiplier-Accelerator Model
+-------------------------------------------------
 
 The previous lecture showed (among other things) how to represent a
-univariate auto-regressive processes as a linear state space model. We
-will do the same for Samuelson's (1939) multiplier-accelerator model.
+univariate auto-regressive processes as a ``LinearStateSpace`` model. We
```

Comment:
Could you also move the second sentence onto it's own line and delete the full stop - thanks